libs/sql-parser/src/main/java/io/crate/sql/tree/Explain.java

```diff
@@ -30,18 +30,22 @@ public class Explain extends Statement {
 
     public enum Option {
         ANALYZE,
-        COSTS
+        COSTS,
+        VERBOSE
     }
 
     private final Statement statement;
     // Possible values for options is `true`, `false`, `null`
     private final Map<Option, Boolean> options;
+    // The below flags exist only for SqlFormatter
     private final boolean analyze;
+    private final boolean verbose;
```

Comment:
I wonder if we could get away with normalizing a explain to a single format when printing, instead of trying to preserve the original format.

No need to change this in this PR, but would be nice if we could eventually get rid of the duplicate information.

Reply:
I'm not sure about the requirements for `SqlFormatter` (how close it should be to the original input), but I'm happy to address this in a separate PR.

Reply:
I think as long as the output is deterministic, and re-parsing the formatted output results in the same structure things should be fine.

> I'm happy to address this in a separate PR.

Would be nice